### DISCUSSION TITLE
?

### DISCUSSION THREAD

**Discussion by trillionstar07** [The-Run-Philosophy-Organization/run#419] [General]

**Comment by xyzhou-1** [comment]
Fake news,next
![image](https://user-images.githubusercontent.com/47747466/170169047-ba7828d7-086b-4a32-bde7-b09bdd10bb92.png)
老图新用

**Comment by XutongJin** [comment]
不明白你们要说啥……美国各种问题又不是一天两天了，这个和润有啥冲突吗

**Comment by AntEffect** [comment]
超过35个了吗？会不会被国务院追责？
